### PR TITLE
revert(chore): broker logs

### PR DESCRIPTION
**Pull request by sc979**
## Description

revert wrongly backported commits

**Fixes** # (none)

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 19.10.x
- [ ] 20.04.x
- [x] 20.10.x
- [ ] 21.04.x (master)

### CODE REVIEW

File: www/include/configuration/configCentreonBroker/formCentreonBroker.ihtml [hunk 1]
```diff
@@ -223,10 +180,13 @@
         });
     }
 
+
+
```
Comment (jeremyjaouen) [inline www/include/configuration/configCentreonBroker/formCentreonBroker.ihtml:184]:
```suggestion
```

File: www/include/configuration/configCentreonBroker/formCentreonBroker.ihtml [hunk 2]
```diff
@@ -223,10 +180,13 @@
...
     function deleteRow(type, id) {
         var row_name = type + '_' + id;
         var parent = document.getElementById(type);
         var delEl = document.getElementById(row_name);
+        //  jQuery("#"+row_name).find('input').qtip('destroy');
```
Comment (jeremyjaouen) [inline www/include/configuration/configCentreonBroker/formCentreonBroker.ihtml:189]:
```suggestion
```

File: www/include/configuration/configCentreonBroker/formCentreonBroker.ihtml (hunk 1 above)
Comment (jeremyjaouen) [inline www/include/configuration/configCentreonBroker/formCentreonBroker.ihtml:184]:
```suggestion
```

**Review by jeremyjaouen — CHANGES_REQUESTED** [review]

File: www/include/configuration/configCentreonBroker/formCentreonBroker.php [hunk 3]
```diff
@@ -38,17 +38,11 @@
     exit();
 }
 
-$id = filter_var(
-    $_REQUEST['id'],
-    FILTER_VALIDATE_INT,
-    ['options' => ['default' => 0]]
-);
-
```
Comment (jeremyjaouen) [inline www/include/configuration/configCentreonBroker/formCentreonBroker.php:46]: must stay

Comment (jeremyjaouen) [inline www/include/configuration/configCentreonBroker/formCentreonBroker.php:46]: must stay

File: www/include/configuration/configCentreonBroker/formCentreonBroker.php [hunk 4]
```diff
@@ -38,17 +38,11 @@
...
 if (
     !$centreon->user->admin
-    && $id !== 0
+    && isset($_GET['id'])
```
Comment (jeremyjaouen) [inline www/include/configuration/configCentreonBroker/formCentreonBroker.php:43]: must stay

File: www/include/configuration/configCentreonBroker/formCentreonBroker.php [hunk 5]
```diff
@@ -38,17 +38,11 @@
...
     && count($allowedBrokerConf)
-    && !isset($allowedBrokerConf[$id])
```
Comment (jeremyjaouen) [inline www/include/configuration/configCentreonBroker/formCentreonBroker.php:51]: mus stay

File: www/include/configuration/configCentreonBroker/formCentreonBroker.php [hunk 6]
```diff
@@ -170,21 +162,23 @@
...
-     */
-    textdomain("help");
-    $nbTabs = count($tabs);
-    for ($i = 0; $i < $nbTabs; $i++) {
-        $tabs[$i]['forms'] = $cbObj->getForms($id, $tabs[$i]['id'], $p, $tpl);
-        $tabs[$i]['helps'] = $cbObj->getHelps($id, $tabs[$i]['id']);
-        $tabs[$i]['nb'] = count($tabs[$i]['forms']);
+} else {
+    if (isset($_GET['id']) && $_GET['id'] != 0) {
+        $id = $_GET['id'];
```
Comment (jeremyjaouen) [inline www/include/configuration/configCentreonBroker/formCentreonBroker.php:167]:
```suggestion
    if ($id !== 0) {
```

**Review by loiclau — APPROVED** [review]

**Review by jeremyjaouen — APPROVED** [review]

**Pull request merged**